### PR TITLE
:closed_lock_with_key: always closed logic

### PR DESCRIPTION
**Pull request by markysoft**
Final changes for opening times

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-64.dev.c2s.nhschoices.net](http://profiles-pr-64.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-64.dev.c2s.nhschoices.net](http://profiles-pr-64.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-64.dev.c2s.nhschoices.net](http://profiles-pr-64.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-64.dev.c2s.nhschoices.net](http://profiles-pr-64.dev.c2s.nhschoices.net)

File: app/lib/parseOpeningTimes.js [hunk 1]
```diff
@@ -50,7 +50,21 @@ function addPadding(parsedTimes) {
   return parsedTimes;
 }
 
+function isOpen(times) {
+  let openDays = 0;
+  daysOfWeek.forEach((day) => {
```
Comment (st3v3nhunt) [inline app/lib/parseOpeningTimes.js]: `forEach` could be replaced with `includes` to prevent the full set being iterated.

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]

File: app/lib/parseOpeningTimes.js (hunk 1 above)
Comment (markysoft) [reply app/lib/parseOpeningTimes.js]: includes won't work as that matches an exact value, rather than the output of a function. Have used 'some' instead.

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-64.dev.c2s.nhschoices.net](http://profiles-pr-64.dev.c2s.nhschoices.net)

File: app/lib/parseOpeningTimes.js [hunk 2]
```diff
@@ -51,14 +51,10 @@ function addPadding(parsedTimes) {
 }
 
 function isOpen(times) {
-  let openDays = 0;
-  daysOfWeek.forEach((day) => {
+  return daysOfWeek.some((day) => {
```
Comment (st3v3nhunt) [inline app/lib/parseOpeningTimes.js]: 👍

**Review by st3v3nhunt — APPROVED** [review]

**Pull request merged**